tests/unit_tests/dft/source/compute_tests.cpp

```diff
@@ -75,7 +75,8 @@ class ComputeTests_real_real_out_of_place_REAL
         }                                                                                 \
         catch (std::exception & e) {                                                      \
             std::string msg = e.what();                                                   \
-            if (msg.find("FFT_UNIMPLEMENTED") != std::string::npos) {                     \
+            if ((msg.find("FFT_UNIMPLEMENTED") != std::string::npos) ||                   \
+                (msg.find("Unimplemented") != std::string::npos)) {                       \
```

Comment:
@s-Nick, was this addition motivated by a new exception that's _not_ a `oneapi::mkl::unimplemented` exception with "Unimplemented" in its message? If yes, could you please share details?